packages/connect/src/ExtensionProvider/ExtensionProvider.ts

```diff
@@ -47,6 +47,17 @@ interface HealthResponse {
   shouldHavePeers: boolean;
 }
 
+function eraseRecord<T> (record: Record<string, T>, cb?: (item: T) => void): void {
```

Comment:
This should be in utils directory. 
Also respective tests should be added for the function

Reply:
I copied it straight from @polkadot/api: https://github.com/polkadot-js/api/blob/c212db26b664a339dc2aa68f6380c40eb72595fb/packages/rpc-provider/src/ws/index.ts#L45-L53

Reply:
As soon as this is now in substrate-connect's codebase it should be placed along the rest of utils, be documented and tested.